src/api.js

```diff
@@ -670,7 +687,7 @@ API.prototype.declare = function(options, handler) {
   if (options.output && options.output !== 'blob') {
     options.output = this._options.schemaPrefix + options.output;
   }
-  this._entries.push(options);
+  this._entries.unshift(options);
```

Comment:
This will end up reversing the whole list of endpoints, right?  I'd rather see a way of including ping at the end while maintaining the order of the others.